src/aging_session/modify_user_session.clj

```diff
@@ -14,5 +14,6 @@
    all existing user sessions."
 
   (terminate-user-sessions
-    [store keys]
-    "Accepts a session-store and multiple keys (session-ids) to terminate the user session(s)."))
+    [store keys reason]
+    "Accepts a session-store, multiple keys (session-ids) to terminate the user session(s) and a reason for why
+     the session was terminated."))
```

Comment:
I know we don't have specs on this protocol, but perhaps being explicit about what a `reason` is could be helpful

Reply:
We are good